Runtime/Input/Interactors/GenericPointer.cs

```diff
@@ -28,6 +28,11 @@ public GenericPointer(string pointerName, IInputSource inputSourceParent)
         {
             if (ServiceManager.Instance.TryGetService<IInputService>(out var inputService))
             {
+                if(inputService.FocusProvider == null)
+                {
+                    throw new ArgumentException($"Couldn't find a valid {nameof(IFocusProvider)} configured for the {nameof(IInputService)}!");
```

Comment:
I think this should be an invalid operation exception rather than an argument exception, since the focus provider is not an argument here?

Reply:
I agree, although my concern now is catching the fact it is not configured.  By handling the exceptions, this point is currently not reached now.

Reply:
Well the excepton is going to cause a pretty nasty red log in the console.